ghzdac/calibrate.py

```diff
@@ -444,11 +444,11 @@ def calibrateDCPulse(cxn,boardname,channel):
     p = scope.packet().\
     select_device(scopeID).\
     reset().\
-    channel(SCOPECHANNEL).\
+    channel(reg.get(keys.SSCOPECHANNEL, 'w', True, 2)).\
     trace(1).\
     record_length(5120).\
     average(128).\
-    sensitivity(Value(100.0,'mV')).\
+    sensitivity(reg.get(keys.SSCOPESENSITIVITYDC, 'v', True, 200*labrad.units.mV)).\
```

Comment:
can omit the pattern, or else I would suggest including units: `v[mV]`. The value should get converted properly, for example if the registry value is in volts instead of millivolts (though we should test that).